src/test/kotlin/no/nav/syfo/behandler/kafka/dialogmeldingfrombehandler/KafkaDialogmeldingFromBehandlerSpek.kt

```diff
@@ -32,7 +43,19 @@ class KafkaDialogmeldingFromBehandlerSpek : Spek({
                     partition,
                 )
                 describe("Happy path") {
```

Comment:
Kan vurdere å legge til en "unhappy path" her, for å teste at man ikke finner partnerreferanse/får oppdatert dialogmeldingEnabled.